Library/Homebrew/formula_versions.rb

```diff
@@ -74,4 +75,20 @@ def bottle_version_map(branch)
     end
     map
   end
+
+  def latest_bottle_rebuild(branch, pkg_version)
```

Comment:
It's probably easier to review this if this is is a modification of the existing function rather than adding a new one?